docs/server/source/drivers-clients/websocket-event-stream-api.rst

```diff
@@ -0,0 +1,105 @@
+The WebSockets Events Stream API
```

Comment:
Good point, I kind of made up "Events Stream API" as I went, but we can change the naming to "Events API" or the file name (or, really, any other option) as we go.